modules/zivid/point_cloud.py

```diff
@@ -62,6 +62,7 @@ def copy_data(self, data_format):
         z:          ndarray(Height,Width)   of float
         rgba:       ndarray(Height,Width,4) of uint8
         bgra:       ndarray(Height,Width,4) of uint8
+        srgb:       ndarray(Height,Width,4) of uint8
```

Comment:
Can you add a test for this in test_point_cloud.py?

Reply:
done